docs/source/settings.rst

```diff
@@ -304,6 +304,8 @@ older file configuration format.
 
 Format: https://docs.python.org/3/library/logging.config.html#logging.config.dictConfig
 
+For more context you can look at the default configuration dictionary for logging, which can be found at ``gunicorn.glogging.CONFIG_DEFAULTS``.
```

Comment:
This is great. I wonder if there's a way we could actually embed `CONFIG_DEFAULTS` here as the default value in the documentation, or if we could link to it.

Reply:
Good question. We can of course link to [logging](https://github.com/benoitc/gunicorn/blob/master/gunicorn/glogging.py) but I don't know if there is any way to auto-link to a variable in the file.

Reply:
It is possible to link directly to a linenumber though. But a commit could then break the location. And locking to a commit hash is also not preferable I believe...